### PR TITLE
22054-added-aria-props-to-buttons

### PR DESCRIPTION
**Pull request by flutistar**
*Issue #:* /bcgov/entity#22054

*Description of changes:*
Added `aria-hidden`, `role`, `aria-label` properties to buttons.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
Very nice.

File: ppr-ui/src/components/tables/common/TableRow.vue [hunk 1]
```diff
@@ -300,6 +304,8 @@
             class="edit-btn"
             color="primary"
             @click="editDraft(item)"
+            aria-hidden="false"
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/common/TableRow.vue]: Is this required on the buttons? I thought they are aria-hidden by default?

File: ppr-ui/src/components/tables/RegistrationTable.vue [hunk 2]
```diff
@@ -237,6 +244,7 @@
                     type="text"
                     label="Submitting Party"
                     density="compact"
+                    aria-hidden="true"
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tables/RegistrationTable.vue]: So the A11Y requirements don't want screen readers to pick up the filtering input fields?

**Review by cameron-eyds — APPROVED** [review]
Looks fine to me! Just some minor questions

File: ppr-ui/src/components/tables/common/TableRow.vue (hunk 1 above)
Comment (flutistar) [reply ppr-ui/src/components/tables/common/TableRow.vue]: I will remove it.

File: ppr-ui/src/components/tables/RegistrationTable.vue (hunk 2 above)
Comment (flutistar) [reply ppr-ui/src/components/tables/RegistrationTable.vue]: I will remove this as well. :)

Thanks

**Pull request merged**